src/start_env/mod.rs

```diff
@@ -417,12 +437,21 @@ fn clean_up() -> impl Future<Item = (), Error = ()> {
                 .collect()
         })
         .then(|_| delete_network())
-        .then(|_| std::fs::remove_dir_all(temp_fs::dir_path()))
-        .map_err(|_| ())
+        .then(|_| {
+            let _ = temp_fs::dir_path().map(std::fs::remove_dir_all);
+            Ok(())
+        })
+        .map_err(|_: ()| ())
```

Comment:
:question: Is it too late to log something here?

Reply:
Like what? Steps to clean-up manually?

Reply:
Not a bad idea actually!